apex-app-archetype/pom.xml

```diff
@@ -50,5 +50,22 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+        <includes>
+          <include>META-INF/maven/archetype-metadata.xml</include>
```

Comment:
The location of this file is determined by the archetype structure. There is no need to filter any other file as Maven resource as those are processed through Velocity.